src/components/ActionChooser.vue

```diff
@@ -17,10 +17,10 @@
     </template>
     <template v-slot:footer>
       <div class="modal-footer">
-        <BaseButton textlike :on-click="abort">
```

Comment:
There will be a lot of those changes.
The BaseButton does not have a required function property anymore. Instead the listeners you declare are passed down and work as if you are working on the button itself.